src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoGetAbbreviatedEraName.cs

```diff
@@ -11,14 +11,15 @@ namespace System.Globalization.Tests
 {
     public class DateTimeFormatInfoGetAbbreviatedEraName
     {
+        private static bool s_isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+
         // PosTest1: Call GetAbbreviatedEraName to get Era's abbreviated name
         [Fact]
         public void PosTest1()
         {
             DateTimeFormatInfo info = new CultureInfo("en-us").DateTimeFormat;
 
-            bool isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
-            string abbreviatedEraName = isWindows ? "AD" : "A";
+            string abbreviatedEraName = s_isWindows ? "AD" : "A";
```

Comment:
would it be a good idea we can have a common method which can be used in different tests that can just return the requested property value? I mean something like 

string abbreviatedEraName = GetPaltformPropertyValue(PlatformProperty.AbbreviatedEraName);

benefit of doing that is we don't have to do s_isWindows everywhere in the code, also if we got more difference in other platforms will be easier to handle. also will be very useful to have one place listing all difference between the platforms. what you think?